docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Dockerfile for `celestia-node`
+
+To build on a M1 Mac:
+```
+docker build --ssh default --platform linux/x86_64 .
+```
+
+To run on a M1 Mac:
```

Comment:
Is there a reason why this defaults to a M1 Mac? 

IMO, the instructions should default to Linux (even though I'm a mac user myself). Can someone with a Linux machine please test the instructions? 

Then we should change the default instructions and put the mac instructions below or a separate section.

Reply:
This is just because I have a M1 Mac and haven’t gone to the trouble of testing on something else.